src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -313,5 +323,117 @@ public static AnnotatedCodeLocationImportanceVersionOne CreateAnnotatedCodeLocat
                     return AnnotatedCodeLocationImportanceVersionOne.Important;
             }
         }
+
+        public interface IBlameHunk
+        {
+            string Name { get; set; }
+
+            string Email { get; set; }
+
+            string CommitSha { get; set; }
+
+            int LineCount { get; }
+
+            int FinalStartLineNumber { get; }
+
+            bool ContainsLine(int line);
+        }
+
+        private class BlameHunk : IBlameHunk
+        {
+            private readonly string _Name;
+            private readonly string _Email;
+            private readonly string _CommitSha;
+            private readonly int _LineCount;
+            private readonly int _FinalStartLineNumber;
+
+            public BlameHunk(string name, string email, string commitSha, int lineCount, int finalStartLineNumber)
+            {
+                _Name = name;
+                _Email = email;
+                _CommitSha = commitSha;
+                _LineCount = lineCount;
+                _FinalStartLineNumber = finalStartLineNumber;
+            }
+
+            public string Name { get => _Name; set => throw new System.NotImplementedException(); }
+            public string Email { get => _Email; set => throw new System.NotImplementedException(); }
+            public string CommitSha { get => _CommitSha; set => throw new System.NotImplementedException(); }
+
+            public int LineCount { get => _LineCount; set => throw new System.NotImplementedException(); }
+
+            public int FinalStartLineNumber { get => _FinalStartLineNumber; set => throw new System.NotImplementedException(); }
+
+            public bool ContainsLine(int line)
+            {
+                if (line >= _FinalStartLineNumber && line <= _FinalStartLineNumber + _LineCount)
+                {
+                    return true;
+                }
+                else
+                {
+                    return false;
+                }
+            }
+        }
+
+        public static IEnumerable<IBlameHunk> ParseBlameInformation(string blameText)
```

Comment:
>ParseBlameInformation [](http://example.com/codeflow?start=46&length=21)

can we test this?

Reply:
OK, let me add a unit test to evaluate just the parser.

Reply:
Added unit test for the git blame text parser. The code for checking region overlap with the results is trivial, so skipping tests for that.